docs/source/notebooks/pcca.ipynb

```diff
@@ -54,9 +80,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "f, ax = plt.subplots(1, 1, figsize=(12, 12))\n",
+    "f, ax = plt.subplots(1, 1, figsize=(12, 5))\n",
     "ax.scatter(*data.T, c=assignments/np.max(assignments))\n",
-    "ax.scatter(*clustering.cluster_centers.T, marker=\"*\", c='red')"
+    "ax.scatter(*clustering.cluster_centers.T, marker=\"*\", c='red', label='Cluster centers');\n",
```

Comment:
I know this is obnoxious but I'd rather have numbers with the cluster indices than stars. I'd probably suggest some kind of rainbow color map too since the proximity of the colors to each other means nothing here

Reply:
you got your cluster indices

Reply:
honored